guides/source/configuring.md

```diff
@@ -1109,6 +1109,13 @@ text/javascript image/svg+xml application/postscript application/x-shockwave-fla
     1. `select=eq(n\,0)+eq(key\,1)+gt(scene\,0.015)`: Select the first video frame, plus keyframes, plus frames that meet the scene change threshold.
     2. `loop=loop=-1:size=2,trim=start_frame=1`: To use the first video frame as a fallback when no other frames meet the criteria, loop the first (one or) two selected frames, then drop the first looped frame.
 
+* `config.active_storage.transformations_for_format` can be used to specify transformations that should be applied to all images of a given content_type.
+
+    ```ruby
+    config.active_storage.transformations_for_format[:jpg] = { saver: { quality: 80, strip: true } }
+    config.active_storage.transformations_for_format[:png] = { saver: { compression: 9, strip: true } }
+    ```
+
```

Comment:
Let's document the `format: "png"` default here (or possibly link to the other place where that's documented, if it's documented elsewhere).

Reply:
The default is mentioned in "8.2 Transforming Images" section of the guide:

```
If a variant is requested, Active Storage will automatically apply transformations 
depending on the image's format:

1. Content types that are variable and not considered web images, will be converted to PNG.
2. If quality is not specified, the variant processor's default quality for the format will be used.
```

Should I link to this?

Reply:
Nah, all good 👍